src/Components/Header/Header.js

```diff
@@ -1,13 +1,190 @@
+            "https://img-cf.kurly.com/shop/data/category/icon_yearend_inactive_pc@2x.1606988604.png",
+          alt: "연말대전",
+          text: "연말대전",
+        },
+      ],
+    };
   }
+
+  componentDidMount() {
+    window.addEventListener("scroll", this.handleScroll);
+  }
+  componentWillUnMount = () => {
+    window.removeEventListener("scroll", this.handleScroll);
+  };
+
+  handleScroll = e => {
+    const scrollTop = ("scroll", window.scrollY);
+    this.setState({ fix: scrollTop > 160 ? true : false });
+  };
+
+  hadnleClick = e => {
+    if (e.target.className === "gnbAllCategory") {
+      this.setState({
+        hoverAction: !this.state.hoverAction,
+        subHoverAction: false,
+      });
+    }
+    if (e.target.className === "gnbOverlay") {
+      console.log("hi");
+      this.setState({
+        hoverAction: false,
+        subHoverAction: false,
+      });
+    }
+  };
+  subMenuMoues = () => {
+    this.setState({
+      subHoverAction: !this.state.subHoverAction,
+    });
+  };
+
   render() {
-    return <></>;
+    return (
+      <>
+        <div
+          className={`${this.state.hoverAction ? "gnbOverlay" : ""}`}
+          onClick={this.hadnleClick}
+        ></div>
+
+        <div className="header">
+          <ul className="headerMenu">
+            <li>
+              <Link to="/">회원가입</Link>
+            </li>
+            <li>
+              <Link to="/">로그인</Link>
+            </li>
+            <li>
+              <Link to="/"> 고객센터</Link>
+            </li>
+            <li>
+              <Link to="/">배송지역 검색</Link>
+            </li>
+          </ul>
+
+          <div className="header__logo">
+            <img src="https://res.kurly.com/pc/service/common/1908/delivery_190819.gif"></img>
+            <img src="./images/marketbully.jpg"></img>
+          </div>
+          <div className={`gnb ${this.state.fix ? "fix" : ""}`}>
+            <ul>
+              <li onClick={this.hadnleClick}>
+                <Link className="gnbAllCategory" to="/">
+                  전체 카테고리
+                </Link>
+              </li>
+
+              {this.state.gnbCategorydata.map(data => {
+                return (
+                  <li key={data.id}>
+                    <Link to="/">{data.content}</Link>
+                  </li>
+                );
+              })}
+            </ul>
+
+            <div className="gnb__search">
+              <input />
+            </div>
+            <div className="gnb__cart">
+              <img src="https://res.kurly.com/pc/ico/1908/ico_cart_x2_v2.png"></img>
+            </div>
+            <div className={`gnbInnerBox ${this.state.subHoverAction ? "hoverd" : ""}`}>
+              <ul>
+                <li>
+                  <Link to="/">
+                    {" "}
+                    <img
+                      src="https://img-cf.kurly.com/shop/data/category/icon_yearend_inactive_pc@2x.1606988604.png"
+                      alt="연말대전"
+                    ></img>
```

Comment:
- img 태그에 들어가는 alt 속성 또한 아래와 같은 이유로 매우 중요합니다.

> 1. 이미지가 로딩되지 않을 경우 alt 속성에 적힌 값이 보여짐.
> 2. 시각장애인의 경우 alt 속성을 통해 어떤 이미지가 보여지는지 알 수 있음.
> 3. SEO 검색엔진 최적화에서 어떤 img인지 인식하는 코드는 alt 속성임.

- 위와 같은 이유로 alt속성은 중요하기 떄문에 src속성보다 먼저 작성하는게 좋습니다.
- 더불어 content가 없는 요소는 self closing 해주세요!
```suggestion
                    <img
                      alt="연말대전"
                      src="https://img-cf.kurly.com/shop/data/category/icon_yearend_inactive_pc@2x.1606988604.png"
                    />
```